### PR TITLE
Added test cases Controller, OpenCDM and Provisioning Test cases

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Added below test cases in this Pull request : 
Framework_Controller_002.test.js
Framework_OpenCDM_001.test.js
Framework_Provisioning_001.test.js

### CODE REVIEW

File: src/framework/Framework_Controller_002.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  getControllerPluginData,
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+} from '../commonMethods/commonFunctions'
+
+export default {
+  title: 'Framework Controller test 02',
+  description: 'Tests basic functionality of the controller module',
+  teardown: restartFramework,
```
Comment (wouterlucas) [inline src/framework/Framework_Controller_002.test.js]: no need

File: src/framework/Framework_Controller_002.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  steps: [
+    {
+      description: 'Get /Service/Controller and validate whether it is Object or not',
```
Comment (wouterlucas) [inline src/framework/Framework_Controller_002.test.js]: Is no longer Get /Service/Controller, but done through jsonRPC.

File: src/framework/Framework_OpenCDM_001.test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  restartFramework,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+export default {
+  title: 'OCDM startup robustness test',
+  description:
+    'Starts and stops the OCDM plugin repeatedly and checks if everything is started correctly',
+  repeat: 30,
+  teardown: restartFramework,
```
Comment (wouterlucas) [inline src/framework/Framework_OpenCDM_001.test.js]: no need

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/framework/Framework_Controller_002.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Controller_002.test.js]: Removed restartFramework

File: src/framework/Framework_Controller_002.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Controller_002.test.js]: Removed restartFramework

File: src/framework/Framework_Controller_002.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Controller_002.test.js]: Renamed the description as I really use json RPC in the test case

File: src/framework/Framework_OpenCDM_001.test.js (hunk 3 above)
Comment (sandeep-vedam) [reply src/framework/Framework_OpenCDM_001.test.js]: Removed restartFramework

**Comment by sandeep-vedam** [comment]
@wouterlucas , Code review comments are addressed for this PR, please take appropriate action if it's Ok

**Pull request merged**